src/www/index.html

```diff
@@ -293,16 +293,33 @@
         <!-- changes-2 removed till here -->
         <!-- changes-2 added from here -->
         <ul style="list-style-type:none;postion: relative;">
-          <li>
-            <input name="session" required="" type="radio" value="single" id="single">
-            <label for="single">Single</label>
+            <li>
+            <label for="single" >
+              <input name="session" type="radio" value="single" id="single" class="form-control-static"> 
+              Single
+            </label>
           </li>
 
           <li>
-            <input name="session" type="radio" value="expand" id="expandable">
-            <label for="expandable">Expandable</label>
+            <label for="expandable">
+              <input name="session" type="radio" value="expand" id="expandable" required>
+              Expandable
+            </label>
           </li>
+          <p id="geeks" style="color: red"></p>
         </ul>
+          <script> 
+            function myFunction() { 
+                var inpObj = document.getElementById("expandable"); 
+                if (!inpObj.checkValidity()) { 
+                    document.getElementById("geeks") 
+                              .innerHTML = inpObj.validationMessage; 
+                } else { 
+                    document.getElementById("geeks") 
+                              .innerHTML = "Input is ALL RIGHT"; 
+                } 
+            } 
+          </script>
```

Comment:
What is the need to add an extra script tag here? The validation should work without it, as well. Pls see to it, Refer how the validation works for Email input field, rest all the fields should follow the same steps.

Reply:
If we don't use the script tag here then the red message will be shown right below the radio button again causing the same issue you mentioned above. 
What I am doing here is show the custom error message right after both the radio buttons instead of showing right after the first radio button which will again cause the misalignment of radio buttons.

Reply:
You have to just show `This field is required` in red at the end of both the radio buttons, as shown by any other fields which are required. If you find an additional script tag used specifically for this purpose, then kindly let me know. I guess, it is better to avoid some extra tags in the index file, when the changes could be implemented without it.

Reply:
"This field is required" won't show below both the fields. It will only show after the first radio button. In the image below you can see that it is showing right after the radio button before the label Single.
![image](https://user-images.githubusercontent.com/42602396/81811832-e294bc80-9542-11ea-817d-22c1ce45e286.png)

Reply:
Yes, and we want this as a result (the one you showed), but without use of script tags. See if it can be implemented, and if not, do let me know.